hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperator.java

```diff
@@ -204,6 +204,9 @@ public Object getOldValue() {
     }
 
     public Data getResult() {
+        if (result == null) {
```

Comment:
Caller should do invalidations on its own end. Returning a not null result from server hides actual issue (also will cause unexpected serialization costs). Do you mind to investigate `NearCachedMapProxyImpl` and apply same logic to `NearCachedClientMapProxy`?

Reply:
Thanks for advice! I updated the code accordingly.